### ISSUE TITLE
Support gettext 0.26.1

### ISSUE THREAD

**Issue by krns** [elixir-cldr/cldr_routes#19]
After upgrading to [gettext 0.26.1](https://hexdocs.pm/gettext/changelog.html#v0-26-0) the following compile time error occurs:

```
app-1       | ** (UndefinedFunctionError) function MyApp.Gettext.dgettext/2 is undefined or private. Did you mean:
app-1       |
app-1       |       * __gettext__/1
app-1       |       * lgettext/4
app-1       |       * lgettext/5
app-1       |       * lngettext/6
app-1       |       * lngettext/7
app-1       |
app-1       |     MyApp.Gettext.dgettext("routes", "de")
app-1       |     lib/my_app_web/router.ex:42: (module)
```

I think this is because we need to adjust the setup of Cldr. 

```elixir
defmodule MyApp.Cldr do
  @moduledoc false

  use Cldr,
    locales: ~w(en de),
    default_locale: "de",
    gettext: MyApp.Gettext,
    providers: [Cldr.Routes]
end
```

Maybe the issue is better off in the Cldr project?

**Comment by kipcole9** [comment]
I've pushed a commit that fixes this issue - at the expense of requiring Gettext 0.26 as a minimum version. Would you consider trying this from GitHub before I release it. Let me know if you see any other issues?

**Comment by krns** [comment]
Thank you so much, @kipcole9! Just tested our application with `main` and can confirm that everything works.

**Comment by krns** [comment]
I will submit a PR that updates the readme.

**Comment by kipcole9** [comment]
I've published [ex_cldr_routes version 1.3.1](https://hex.pm/packages/ex_cldr_routes/1.3.1) with the following changelog entry:

### Bug Fixes

* Requires [Gettext version 0.26](https://hex.pm/packages/gettext/0.26.0) or later since that version has an incompatible API with previsous versions and we wish to avoid more conditional code. Thanks to @krns for the PR to align the documentation and examples as well.

Thanks again for the issue, and the PR.
